plugin.json

```diff
@@ -2,7 +2,7 @@
     "id": "com.mattermost.apps",
     "name": "Apps",
     "description": "This plugin powers the Mattermost Apps Framework and is required for any Apps to run",
-    "homepage_url": "https://github.com/mattermost/mattermost-plugin-apps",
+    "homepage_url": "https://mattermost.com/marketplace/app-framework",
```

Comment:
Given that this plugin currently targets developers should we link to https://developers.mattermost.com/integrate/apps/ instead to make it easier to get started?

Reply:
0/5 @hanzei yes this is aimed at developers. But we're showing this to everyone and Aaron's page has a broader context. 

Also the related PR must go to the marketplace very soon. I propose we can open a follow-up PR for further tweaks in the next day or two so the URLs stay consistent between the repo and the marketplace.

Reply:
Makes sense to me